### ISSUE TITLE
NVDA fails to move focus to the selected profile in the Firefox profile selection dialog when it is focused for the first time

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#10906]
### Steps to reproduce:
1. Create more than one  profile in Firefox, and uncheck "use the selected profile without asking at startup".
2. Close and reopen Firefox.
3. Observe what is being read when the dialog gains focus - try to check which profile is selected with NVDA+up arrow or NVDA+Tab.

### Actual behavior:
When dialog gains focus only text of the dialog is announced. Reading current line reads blank,, and attempting to read object with focus announces frame read only.
### Expected behavior:
Currently selected profile should be announced in all above cases - when  focus is moved from the initial place and back this works as expected.
#### NVDA installed/portable/running from source:
Portable
#### NVDA version:
2020.1 Beta 1
#### Windows version:
Windows 7 X64
#### Name and version of other software in use when reproducing the issue:
Nightly 76
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The same problem occurs with installed 2019.2.1. I do not think this is a regression.
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
N/A

I've opened this as an NVDA issue because JAWS works correctly in this situation. 

cc @jcsteh @MarcoZehe

**Comment by Adriani90** [comment]
@lukaszgo1 are you seeing this problem on Windows 8.1 or higher with Firefox 123?
